api/query/query.go

```diff
@@ -61,6 +63,26 @@ func (qh queryHandler) processInput(w http.ResponseWriter, r *http.Request) (*sh
 	return &filters, testRuns, summaries, nil
 }
 
+func (qh queryHandler) validateSummaryVersions(v url.Values) (bool, error) {
+	filters, err := shared.ParseQueryFilterParams(v)
+	if err != nil {
+		return false, err
+	}
+	testRuns, filters, err := qh.getRunsAndFilters(filters)
+	if err != nil {
+		return false, err
+	}
+
+	for _, testRun := range testRuns {
+		summaryURL := shared.GetResultsURL(testRun, "")
+		// All new summary URLs end with "-summary_v2.json.gz".
+		if !strings.HasSuffix(summaryURL, "-summary_v2.json.gz") {
+			return false, nil
```

Comment:
Nit: Add a logging here for debugging purpose. Print out the file format

Reply:
Added 👍